2.evaluate_data/compute_sc_anomalyze_data/nbconverted/compute_sc_feature_importance_data.py

```diff
@@ -128,7 +130,7 @@
                     str(anomalyze_models_path / anomaly_model_name),
                     str(treatment_feature_importances_path),
                     str(plate_mapping_path),
-                    str("compute_feature_importance_by_plate.py"),
+                    str("nbconverted/compute_feature_importance_by_plate.py"),
```

Comment:
is the `str()` necessary here? Since what you are providing are strings already. It could be I might be missing something here.